File: Code/Api/Stocky.Model/Admin/OrderModel.cs
using System;
using System.Collections.Generic;
using static Stocky.Common.Enums;

namespace Stocky.Model.Admin
{
    public class OrderModel
    {
        public long Id { get; set; }
        public DateTime OrderDate { get;  set; }
        public decimal TotalAmount { get;  set; }
        public OrderStatus OrderStatus { get;  set; }
        public List<OrderItemModel> OrderItems { get;  set; }
        public long CustomerId { get; set; }
    }
}

File: Code/Admin/stocky/src/app/module/order/model/orderModel.ts
import { OrderStatus } from '../../core/enums/orderStatus';
import { OrderItemModel } from './orderItemModel';

export class OrderModel {
    id: number;
    orderDate: Date;
    totalAmount: number;
    orderStatus: OrderStatus;
    orderItems = new Array<OrderItemModel>();
    customerId: number;

    constructor() {
        this.orderItems = new Array<OrderItemModel>();
    }
}



